src/Nethermind/Nethermind.JsonRpc.Test/Modules/Eth/EthRpcModuleTests.EstimateGas.cs

```diff
@@ -59,7 +59,7 @@ public async Task Eth_estimateGas_web3_sample_not_enough_gas_system_account()
                 "{\"gasPrice\":\"0x100000\", \"data\": \"0x70a082310000000000000000000000006c1f09f6271fbe133db38db9c9280307f5d22160\", \"to\": \"0x0d8775f648430679a709e98d2b0cb6250d2887ef\"}");
             string serialized =
                 ctx._test.TestEthRpc("eth_estimateGas", ctx._test.JsonSerializer.Serialize(transaction));
-            Assert.AreEqual("{\"jsonrpc\":\"2.0\",\"result\":\"0x5898\",\"id\":67}", serialized);
+            Assert.AreEqual("{\"jsonrpc\":\"2.0\",\"result\":\"0x53b8\",\"id\":67}", serialized);
```

Comment:
Why did we change the result? Could you explain?

Reply:
In TestRpcBlockchain we changed spec provider from Mainnet.Instance to Berlin, this changed gas usage (and thus block hashes). We were mining low block numbers -> hence using Frontier.